environment/src/lib.rs

```diff
@@ -55,7 +55,7 @@ pub trait Environment: 'static + Clone + Debug + Default + Send + Sync {
     /// The list of beacon nodes to bootstrap the node server with.
     const BEACON_NODES: &'static [&'static str] = &[];
     /// The list of sync nodes to bootstrap the node server with.
-    const SYNC_NODES: &'static [&'static str] = &["127.0.0.1:4135"];
```

Comment:
Please revert this, it breaks development guide instructions.

Reply:
happy to revert it for now, but we should probably avoid this interfering with tests; I can propose an adjustment that doesn't affect the dev mode separately :+1:

Reply:
done in https://github.com/AleoHQ/snarkOS/pull/1708